### PR TITLE
doc: README

### PR DESCRIPTION
**Pull request by Ziqi-Yang**

### CODE REVIEW

**Review by eval-exec — APPROVED** [review]
Thank you! :blue_heart:

File: eee.el [hunk 1]
```diff
@@ -139,7 +139,7 @@ NAME is passed to `ee-start-terminal-function'."
 	 (target-file (string-trim target-file)))
     (when (not (string-empty-p target-file))
       (message "ee-yazi opening: %s" target-file)
-      (ee-find-file target-file))))
+      (ee-find-file target-file))))q
```
Comment (eval-exec) [inline eee.el:142]: It seems that the `q` should be removed. @Ziqi-Yang

Comment (Ziqi-Yang) [reply eee.el:142]: Sry

Comment (Ziqi-Yang) [reply eee.el:142]: Can you do it? I see you have merged the commit into `main` branch

Comment (eval-exec) [reply eee.el:142]: No worries! I'll fix it. Thanks for the contribution!

**Pull request merged**